uptime_service_validation/coordinator/server.py

```diff
@@ -69,16 +73,58 @@ def setUpValidatorPods(time_intervals, logging, worker_image, worker_tag):
         keyspace = os.environ.get("AWS_KEYSPACE")
         start_timestamp = datetime_formatter(mini_batch[0]).replace(" ", "\ ")
         end_timestamp = datetime_formatter(mini_batch[1]).replace(" ", "\ ")
-
+        
+        # Unique names are needed for the configmaps
         configmap_name_suffix = str(
             "".join(random.choices(string.ascii_lowercase + string.digits, k=5))
```

Comment:
Does this mean that every time a job will run, a new configmap will be created?
In term of content, what distinguishes one ConfigMap from another?

Reply:
The configmap holds the entrypoint to the job. Because the command to each job differs (they are deployed to check different time intervals of the submissions) each entrypoint differs, thus each configmap is different.

Unfortunately we cannot just define the configmap in the python code and use it, we have to create it aswell. I gave all of them unique names and put some lines of code that should delete the configmaps after the jobs are done.

Only one configmap would be nice, but I do not think this situation permits that.